### PR TITLE
Added restrictions for what can be inserted into most belts

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

This may need a lot of tweaking. Basically I just went through and added what I thought made sense.

**Changelog**

:cl: Swept
- add: Most belts now have restrictions for what can be inserted into them

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -242,27 +287,29 @@
...
           tags:
           - BotanyHatchet
-      hydro:
-        whitelist:
-          tags:
-          - PlantAnalyzer # Dunno what to put here, should be aight.
+      # hydro:
+      #   whitelist:
+      #     tags:
+      #     - PlantAnalyzer # Dunno what to put here, should be aight.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:312]: why commented out?

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml [hunk 2]
```diff
@@ -72,25 +74,31 @@
...
-    #     - Painter
-    #     - GeigerCounter
-    #     - Flare
-    #     - Coilsofwire
-    #     -
-    #     -
-    #   components:
-    #     - SignalLinker
     capacity: 100
+    # TODO: Fill this out more.
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:78]: is this a TODO for this PR or for future PRs

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml (hunk 1 above)
Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Clothing/Belt/belts.yml:312]: PlantAnalyzer isn't a thing yet

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml (hunk 2 above)
Comment (SweptWasTaken) [reply Resources/Prototypes/Entities/Clothing/Belt/belts.yml:78]: Future PRs

**Comment by SweptWasTaken** [comment]
Passing tests

**Comment by SweptWasTaken** [comment]
Basically this is DNM since all the tags need to be refactored to instead of defining what is spawned with SpawnOnXBelt tags.

**Comment by SweptWasTaken** [comment]
This is actually ready to be merged, refactoring to SpawnOnXBelt would break the mapped storage visualizers and there's no way around that since the visualizer uses whitelist and needs a specific tag.

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**